action.yml

```diff
@@ -28,6 +28,18 @@ inputs:
   resource-group-name:
     description: 'Enter the resource group name of the web app'
     required: false
+  type:
+    description: 'Enter deployment type (JAR, WAR, EAR, ZIP, Static)'

```

Comment:
Are "lib/startup" types are also supported from here. If yes, can you add them in the description?
Reference: https://learn.microsoft.com/en-us/azure/app-service/deploy-zip?tabs=cli#kudu-publish-api-reference

Reply:
We are trying to phase out usage of lib/startup so we will not be including them here